### PR TITLE
Update composer to install concrete5/core ^9.0.1

### LINKED ISSUE #39: Version 9 support
Version 9 has a few issues that prevent us from upgrading by default at the moment due to PHP 8 incompatibility

- `theorchard/monolog-cascade`: No PHP 8 support, blocks install
- `theorchard/monolog-cascade`: Dev dependency, requires `minimum-stability: dev`
- `laminas/cache`: Laminas includes a ton of cache adapters that are abandoned and don't support PHP 8, [they have a recommended fix](https://docs.laminas.dev/laminas-cache/installation/#avoid-unused-cache-adapters-are-being-installed). Not sure yet if that should go in the core or in this repo.

### PR DESCRIPTION
**Pull request by Ruud-Zuiderlicht**
Adds version 9 support, fixes #39

### CODE REVIEW

**Comment by aembler** [comment]
This looks good to me, but I'll let @KorvinSzanto make the final call.

File: composer.json [hunk 1]
```diff
@@ -19,19 +19,14 @@
   },
   "require": {
     "composer/installers": "^1.3",
-    "concrete5/core": "^8.2",
+    "concrete5/core": "^9.0.0",
```
Comment (KorvinSzanto) [inline composer.json]: Can you change this to `^9.0.1`? I'd also consider keeping ^8.2 but we should probably just go 9.0.1 minimum and let people change the requirement if they really need to.

**Review by KorvinSzanto — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Ruud-Zuiderlicht) [reply composer.json]: If you keep 8.2 as an option, will that work with `doctrine/dbal` and `concrete5/phpass-compat` requirements removed?

**Review by KorvinSzanto — APPROVED** [review]

**Comment by KorvinSzanto** [comment]
Thanks @Ruud-Zuiderlicht, I've tagged 1.3.0 with this PR included.

**Pull request merged**